src/instructions/proctored_exam/ReadyToStartProctoredExamInstructions.jsx

```diff
@@ -13,7 +13,7 @@ const ReadyToStartProctoredExamInstructions = () => {
     startProctoredExam,
   } = state;
   const { attempt, reviewPolicy } = exam;
-  const { total_time: examDuration } = attempt;
+  const examDuration = attempt.total_time ? attempt.total_time : exam.total_time;
```

Comment:
I realized that `attempt.total_time` and `exam.total_time` can return different values, as `attempt.total_time` takes resumed attempts into account (meaning the attempt's total time could be less depending on whether or not it is part of a second change attempt). 

I updated this assignment to default to the exam total time in the case where `attempt.total_time` doesn't exist, to solve for edx-exams.